Makefile

```diff
@@ -64,7 +64,7 @@ publish:
 	make test
 	# not using lerna independent mode atm, so only update packages that have changed since we use ^
 	# --only-explicit-updates
-	./node_modules/.bin/lerna publish --npm-tag=next --exact
+	./node_modules/.bin/lerna publish --npm-tag=next --exact --skip-temp-tag
```

Comment:
doesn't seem to be necessary now that Logan's PR is in (topo sort publish), also makes publish a million times faster